build.gradle

```diff
@@ -69,5 +69,5 @@ subprojects {
 }
 
 ext {
-    fluxCVersion = 'd379bc018621e33ca1cfc912bf6eb88438d8f534'
+    fluxCVersion = '7c3b65d0eccc6acc5a55017b10a8fa1bfa19b8f2'
```

Comment:
I still don't remember our FluxC version update process -> do we want to merge this to develop? Or should we release a new version of FluxC first?

Reply:
The FluxC commit is already in develop so this should be fine